### DISCUSSION TITLE
Application consistently rejected: I am not in the country where the school is located

### DISCUSSION THREAD

**Discussion by sylviaD25** [community/community#146792] [GitHub Education]
### Hi everyone,

School is currently on break and I'm not in the country where the school is. My submissions have been rejected every time, all for the same reason:The school you selected does not appear to have a campus location in your country. What should I do?

**Comment by davevad93** [comment]
Hi @sylviaD25 ,

In your case you'll have to wait until you come back from the school break, unfortunately it's not possible to apply if you are in another country since it's necessary to activate the geolocation in order to submit the application and if your location doesn't match with that of your campus, the application will be rejected.
